test/aem/aemupload.test.js

```diff
@@ -27,7 +27,7 @@ describe('AEM Upload', function() {
         nock.cleanAll();
     });
 
-    it('AEM upload smoke test', async function() {
+    it('AEM upload smoke test', async function() { // TypeError: Cannot read property 'obtainBuffer' of null
```

Comment:
why the comment?

Reply:
Leftover from writing the code. Removed.